docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -23,6 +23,38 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2021.01.02 to 2021.02.00
 
+### Theme updates and CSS variables
+
+The theme of MapStore has been updated to support CSS variables to some aspect of the style in particular colors and font families.
+The `web/client/themes/default/variables.less` file contains all the available variables described under the `@ms-theme-vars` ruleset.
+It suggested to update the lesscss variables in the projects because the variables starting with `@ms2-` will be deprecated soon:
```

Comment:
```suggestion


The theme of MapStore has been updated to support CSS variables for some aspects of the style, in particular colors and font families.
The `web/client/themes/default/variables.less` file contains all the available variables described under the `@ms-theme-vars` ruleset.
It is suggested to : 

- update the lesscss variables in the projects because the variables starting with `@ms2-` will be deprecated soon:
```